hack/env.sh

```diff
@@ -7,16 +7,20 @@ if [ -z $SKIP_VAR_SET ]; then
         export NETWORK_RESOURCES_INJECTOR_IMAGE=${NETWORK_RESOURCES_INJECTOR_IMAGE:-ghcr.io/k8snetworkplumbingwg/network-resources-injector}
         export SRIOV_NETWORK_CONFIG_DAEMON_IMAGE=${SRIOV_NETWORK_CONFIG_DAEMON_IMAGE:-ghcr.io/k8snetworkplumbingwg/sriov-network-operator-config-daemon}
         export SRIOV_NETWORK_WEBHOOK_IMAGE=${SRIOV_NETWORK_WEBHOOK_IMAGE:-ghcr.io/k8snetworkplumbingwg/sriov-network-operator-webhook}
+        export SRIOV_NETWORK_METRICS_EXPORTER_IMAGE=${SRIOV_NETWORK_METRICS_EXPORTER_IMAGE:-ghcr.io/k8snetworkplumbingwg/sriov-network-metrics-exporter}
         export SRIOV_NETWORK_OPERATOR_IMAGE=${SRIOV_NETWORK_OPERATOR_IMAGE:-ghcr.io/k8snetworkplumbingwg/sriov-network-operator}
+        export KUBE_RBAC_PROXY_IMAGE=${KUBE_RBAC_PROXY_IMAGE:-"gcr.io/kubebuilder/kube-rbac-proxy:v0.15.0"}
 else
         # ensure that OVS_CNI_IMAGE is set, empty string is a valid value
         OVS_CNI_IMAGE=${OVS_CNI_IMAGE:-}
+        KUBE_RBAC_PROXY_IMAGE=${KUBE_RBAC_PROXY_IMAGE:-}
```

Comment:
I am not sure this is right here.
we don't support deploying the metrics exporter without the rbac proxy right?

Reply:
right, but if a user deploys with `SKIP_VAR_SET=true`, he will get an error even if he doesn't want to use the metrics exporter. As this is an optional feature, I wouldn't break any existing deployment. WDYT?